datasets/librispeech/_metadata.json

```diff
@@ -0,0 +1,254 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/",
```

Comment:
I'm getting 4 errors and 2 warnings when validating this example, partly due to missing context such as 
```
    "includes": "ml:includes",
    "recordSet": "ml:RecordSet",
```